test/suites/container_devices_nic_bridged_filtering.sh

```diff
@@ -203,6 +205,25 @@ test_container_devices_nic_bridged_filtering() {
       false
   fi
 
+  # Add a fake IP within ipv4.routes range
+  lxc exec "${ctPrefix}A" -- ip a flush dev eth0
+  lxc exec "${ctPrefix}A" -- ip a add 10.21.0.138/8 dev eth0
```

Comment:
@tomponline I think I've just realised why this isn't working - this IP is outside the range for the network that container A's eth0 device is a child of (https://github.com/lxc/lxd/pull/9953/files#diff-1f0a251e7be7b83cffcce0dca5ce41675fefc0fbb926d78a56f7234c77b685edR38).

So I'll need to make another network... but if I do that then that ping will surely fail unless I also configure peering.

Perhaps I can broaden the original network and then keep all the subnets within it?

Reply:
The tests are now working for xtables with IPv4. Working on tests for IPv6 now.